lib/sinon/default-behaviors.js

```diff
@@ -269,6 +269,7 @@ var defaultBehaviors = {
         Object.defineProperty(rootStub.rootObj, rootStub.propName, {
             value: newVal,
             enumerable: true,
+            writable: true,
```

Comment:
I assume the build failed without this change on node 18. Do you know why?

Reply:
It did.

There was an error about changes to `process.env` using `defineProperty` have to be `writable` (and other things).